src/widgets/dialogs/EmotePopup.cpp

```diff
@@ -448,6 +449,22 @@ void EmotePopup::loadChannel(ChannelPtr channel)
     }
 }
 
+bool EmotePopup::eventFilter(QObject *object, QEvent *event)
+{
+    if (object == this->search_ && event->type() == QEvent::KeyPress)
+    {
+        QKeyEvent *keyEvent = static_cast<QKeyEvent *>(event);
+        if (keyEvent->key() == Qt::Key_Backspace &&
+            keyEvent->modifiers() == Qt::ControlModifier &&
+            this->search_->selectionLength() > 0)
```

Comment:
```suggestion
        if (keyEvent == QKeySequence::DeleteStartOfWord &&
            this->search_->selectionLength() > 0)
```

We should be platform-agnostic.